### DISCUSSION TITLE
Antimatter Chemistry崩溃

### DISCUSSION THREAD

**Discussion by iPodtouch6557** [Meloong-Git/PCL#2269] [提问] [answered]
我在设置里设置为了自动选择合适的Java版本，但是旧版启动成功，新版直接崩溃，需要强行指定一个jre8才能启动
截屏：
![image](https://github.com/Hex-Dragon/PCL2/assets/118656946/b1a82136-c768-486d-917f-62099c2e3c9f)
错误报告：
[错误报告-2023-7-17_11.49.05.zip](https://github.com/Hex-Dragon/PCL2/files/12065977/-2023-7-17_11.49.05.zip)
整合包文件：
旧版
[Antimatter Chemistry.zip](https://github.com/Hex-Dragon/PCL2/files/12065994/Antimatter.Chemistry.zip)
[原始整合包.zip](https://github.com/Hex-Dragon/PCL2/files/12065997/default.zip)
新版
[Antimatter chemistry 1.4.2.zip](https://github.com/Hex-Dragon/PCL2/files/12065999/Antimatter.chemistry.1.4.2.zip)
我在隔壁整合包作者那里也交了issue

**Comment by SALTWOOD** [comment, marked as answer]
也许可以等下一个版本，#2233 和#2194 似乎都是和你的问题有关
可以等快照或正式版发布再看看

**Reply by iPodtouch6557** [reply]
那为啥旧版本不会崩溃呢，我今天还用旧版启动了一下子

**Reply by SALTWOOD** [reply]
估计和模组加载器版本有关（？
参考#2233

**Reply by iPodtouch6557** [reply]
那么等新版本出来了再看看，先不要mark answer了

**Reply by WForst-Breeze** [reply]
超时关闭，如有补充请继续评论

**Reply by iPodtouch6557** [reply]
问题已解决，忘了来说了qwq
